modules/virt-storage-wizard-fields-web.adoc

```diff
@@ -22,9 +22,15 @@
 |Interface
 |Type of disk device. Supported interfaces are *virtIO*, *SATA*, and *SCSI*.
 
-|Storage class
+|Storage Class
 |The `StorageClass` that is used to create the disk.
 
+|Advanced > Volume Mode
+|Defines whether the persistent volume uses a formatted filesystem or raw block state. Default is *Filesystem*.
+
+|Advanced > Access Mode
```

Comment:
Should these instances of `>` be `->`?

s/Default is/The default Volume Mode is